### PR TITLE
Notifications. First Try

### PR DESCRIPTION
**Pull request by bevekashaivelt**
I think notifications are more or less ready, it still needs to be tested though.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -4,4 +4,18 @@
     <string name="file_chooser"></string>
     <string name="file_chooser_default"></string>
     <string name="file_chooser_custom"></string>
+    <string-array name="notification_settings">
+        <!--item>Every minute</item>
+        <item>Once in 5 minutes</item>
+        <item>Once in 10 minutes</item-->
```
Comment (mayim-chayim) [inline app/src/main/res/values/strings.xml:10]: why is this removed?

**Review by KFMDMSolutions — APPROVED** [review]

**Review by KFMDMSolutions — APPROVED** [review]

**Review by KFMDMSolutions — APPROVED** [review]

**Comment by KFMDMSolutions** [comment]
image

to where does this refer too

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (bevekashaivelt) [reply app/src/main/res/values/strings.xml:10]: WorkManager can run only once in 15 minutes, any faster would require a foreground service. There could be some issues when the user plugs in the device

**Comment by KFMDMSolutions** [comment]
> I think notifications are more or less ready, it still needs to be tested though.

was this line of code replaced or removed  intentionally?

 ActivityCompat.requestPermissions(this, new String[]{
                Manifest.permission.READ_EXTERNAL_STORAGE,
                Manifest.permission.CAMERA,
                Manifest.permission.READ_EXTERNAL_STORAGE,
                Manifest.permission.WRITE_EXTERNAL_STORAGE

        }, 0);

**Comment by bevekashaivelt** [comment]
> > I think notifications are more or less ready, it still needs to be tested though.
> 
> was this line of code replaced or removed intentionally?
> 
> ActivityCompat.requestPermissions(this, new String[]{
> Manifest.permission.READ_EXTERNAL_STORAGE,
> Manifest.permission.CAMERA,
> Manifest.permission.READ_EXTERNAL_STORAGE,
> Manifest.permission.WRITE_EXTERNAL_STORAGE
> 
> ```
>     }, 0);
> ```

No

**Comment by bevekashaivelt** [comment]
Might be better to use the other pull request.
